cumulus_library/studies/core/core_templates/diagnosticreport.sql.jinja

```diff
@@ -4,8 +4,6 @@
 -- This table includes all fields of interest to the US Core DiagnosticReport profiles.
 -- EXCEPT FOR:
 -- * the 'presentedForm' field, which is an attachment array that is stripped out by the ETL.
--- * the `reporter` field, simply due to it not likely being interesting to consumers
---   and being an array field, which would require a lot of row duplication.
```

Comment:
This line should have said `performer` not `reporter` - and the math on interest-to-consumers has changed now that we have actual practitioners in the database. It still requires a lot of row duplication, but...